lib/gherkin_lint/linter/tag_constraint.rb

```diff
@@ -0,0 +1,32 @@
+module GherkinLint
+  # Mixin to lint for tags that have certain string contraints
+  module TagConstraint
```

Comment:
I reread this and I think what you are asking for is an implementation of RequiredTagMatchesRegexp (@mcc[\d]). At this point, I'm not prepared to implement because I want to to limit the scope of this PR, but I will gladly attempt an implementation once this is merged. Not sure how the ```Skip``` would work?  What are the requirements.  I believe this would be a third linter.